docs/book/getting_started/interoperability.md

```diff
@@ -0,0 +1,27 @@
+# Interoperability
+
+## Bootstrapping
+
+Bootstrapping is the process in which:
+
+1. A cluster is bootstrapped
+1. A machine is bootstrapped and takes on a role within a cluster
+
+
+[CAPBK](https://github.com/kubernetes-sigs/cluster-api-bootstrap-provider-kubeadm) is the reference bootstrap provider and is based on `kubeadm`. CAPBK essentially codifies the steps for [creating a cluster](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) in multiple configurations.
+
+See [Machine States Bootstrapping CAEP](https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20190610-machine-states-preboot-bootstrapping.md) for the various states that machines go through while bootstrapping.
+
+
+### Certificate Management
+
+CAPBK will generate the certificates and keys used for bootstrapping a cluster if they don't exist.  CAPBK will also reuse existing certificates provided they follow this convention:
+
+* Each certificate must be stored in a single secret named one of:
+    * `<cluster name>-ca`
+    * `<cluster name>-etcd`
+    * `<cluster name>-proxy`
+    * `<cluster name>-sa`
+* Each secret must have the following label:
+  * `cluster.x-k8s.io/cluster-name=<cluster name>`
+* Secrets must be of type TLS with 2 values `tls.crt` and `tls.key`
```

Comment:
Should we specify their encoding format (e.g. PEM)? Is it worth having a dummy example in here?